retrofit/src/main/java/retrofit/OkHttpCall.java

```diff
@@ -30,6 +30,7 @@
 
   private volatile com.squareup.okhttp.Call rawCall;
   private boolean executed; // Guarded by this.
+  private boolean canceled; // Guarded by this.
```

Comment:
Can this just be a volatile? We never read and write in one place. One person just writes and another just reads.